### ISSUE TITLE
Changes to individual not saved

### ISSUE THREAD

**Issue by mkroon1** [LOVDnl/LOVD3#192]
For one individual (74490 on the shared installation), changes to the diseases linked to that individual are unchangable. LOVD claims success after the user's action when only one disease (CORD) is selected from the list and the form is submitted. The log states:

~~~
	Event	2017-02-22 16:44:10	M. Kroon	IndividualEdit	Delete	Disease entries successfully removed from individual 00074490
	Event	2017-02-22 16:44:10	M. Kroon	IndividualEdit	Delete	Edited individual information entry 00074490
~~~

Yet the original list of diseases is displayed.

Note: this individual has 2027 diseases linked, which is presumably a mistake.

**Comment by mkroon1** [comment]
_TL;DR:  I propose not to fix this._

Bumped down the severity of this bug, as it appears to be only affecting this individual, only due to the presumed mistake and there is a work-around.

The problem is caused by the usage of `GROUP_CONCAT()` in the individual object's load-entry SQL query. The default 1024 character limit restricts the number of links to diseases that can be deleted to about 170 at a time.

One solution is to add a line `$this->sSQLPreLoadEntry = 'SET group_concat_max_len = 200000';` which is used for other objects which join to large numbers of genes for example.

However, this is clearly a mistake and shouldn't happen. The individual with the second highest number of linked diseases has only 7 of them. Unless there is a specific use case where individuals get a high number of diseases linked, I propose not to add the query setting `group_concat_max_len`.

The deletion of diseases still works, only about 170 at a time. I will apply the change in the shared installation until only the one (CORD) disease is left.

**Comment by ifokkema** [comment]
Fixed anyway in #232. It's better to take care of this, as the `group_concat_max_len` is now assumed. We may have individuals with only max 7 diseases, but other database may have more and shorted max lengths at the same time. It's just a one-line fix, so I implemented it.
